### ISSUE TITLE
[a11y] Issue 42: "Closed" badge should be read separately from the title

### ISSUE THREAD

**Issue by kk-slider18** [NewJerseyStyle/polis#138]
**Description:**
Currently, the screen reader reads the Closed badge together with the title, making it harder for users to distinguish between them. The badge should be announced separately to improve clarity and accessibility.

**Component/Area:** Closed Page

**Source:** Screen Reader

**Affected Elements:** titles, badges

**Comment by Natasha1110** [comment]
Could you provide link to the space being mentioned here so I can see the break. From my understanding, this would be a code fix and wouldn't be UX.

**Comment by YumiChen** [comment]
Was checking this page:
https://civictech-polis.azurewebsites.net/4xafbcyvkz

Currently title and closed badge would be announced in one sentence for the badge is placed within the `h1` tag:
![Image](https://github.com/user-attachments/assets/04edbaaa-e52a-4b46-b087-d99f744d28bc)
![Image](https://github.com/user-attachments/assets/a5cdc99d-5af9-43b9-aaaf-da19989f0719)

I think I'll move the closed badge out of `h1` so that they'll be announced in separate sentences
